fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -22,31 +24,31 @@ def self.run(params)
         end
 
         commands = []
-        commands << Fastlane::Actions.sh("security create-keychain -p #{escaped_password} #{keychain_path}", log: false)
+        commands << Fastlane::Actions.sh("security create-keychain -p #{escaped_password} #{keychain_path}", log: should_log)
```

Comment:
I'd recommend using `Fastlane::Actions.sh_control_output(command, print_command: false, print_command_output: true)` here, just to always bias towards not logging the password.

Feel free to prepend that with a special call to `UI.command(command)` with the password parameter removed (when the  env var is present) if you want to get the rest of the command logged.